types/twine-sugarcube/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
You cannot add any rules here. Please fix the error by removing this. If you really need to disable error somewhere, do it inline for that line instead of disabling for whole package